starter/js/atm.js

```diff
@@ -1,31 +1,64 @@
 //Begin with the document ready function
+$(document).ready(function() {
+ //Checking account deposit function
+ //On click of the depositChecking button
 
-    //Checking account deposit function
+ $("#depositChecking").click(function(event) {
```

Comment:
Indent all of these listeners once (2 or 4 spaces, depending on your formatting) – not just one space. Other than that nit-picky note, nice job with the code formatting!